dashboard/test/models/course_version_test.rb

```diff
@@ -10,4 +10,67 @@ class CourseVersionTest < ActiveSupport::TestCase
     assert_instance_of Script, course_version.content_root
     assert_equal course_version, course_version.content_root.course_version
   end
+
+  # One "integration test" of seeding from DSL creating a CourseVersion for a Script with is_course true.
+  # Other cases are covered by directly testing update_course_version below.
+  test "Script.setup creates CourseVersion if is_course is true" do
+    script_file = File.join(self.class.fixture_path, 'test-script-course-version.script')
+    scripts, _ = Script.setup([script_file])
+    script = scripts.first
+
+    course_version = CourseVersion.find_by(key: 'xyz-1234')
+    assert_equal course_version, script.course_version
+    assert_equal 'xyz-1234', course_version.key
+    assert_equal '1234', course_version.display_name
+  end
+
+  test "update_course_version creates CourseVersion for script that doesn't have one if is_course is true" do
```

Comment:
it looks like you updated the name of update_course_version method so should it get updated in the name of the tests?